gst/isomp4/qtdemux.c

```diff
@@ -8554,6 +8554,13 @@ gst_qtdemux_guess_bitrate (GstQTDemux * qtdemux)
     return;
   }
 
+  /* Check that the size is valid */
+  if (size < 0) {
+    GST_DEBUG_OBJECT (qtdemux,
+        "Size in bytes of the stream not valid - bailing");
```

Comment:
I wouldn't call it "not valid" , the duration is "infinite". So, eather it's a "not known" eather "size is negative", but it's valid, isn't it (it's a dialectical issue, can size of the stream be invalid ?). Also would be nice to put the value of the size into the log, to know is it (-1) that is infinite, or is it a garbage from the bug